src/test/java/software/amazon/awssdk/crt/test/HttpRequestResponseTest.java

```diff
@@ -171,44 +178,65 @@ private boolean shouldRetry(TestHttpResponse response) {
         return false;
     }
 
-    public TestHttpResponse testRequest(String method, String endpoint, String path, String requestBody, int expectedStatus) throws Exception {
+    public TestHttpResponse testRequest(String method, String endpoint, String path, String requestBody, boolean useChunkedEncoding, int expectedStatus) throws Exception {
```

Comment:
trivial: just add an overload signature for this function that is the old one, which calls this overload with `useChunkedEncoding` set to `false`, much smaller change.

Reply:
Seems to just make it less clear and explicit to me, but doesn't shave off the majority of the code I needed to change.